bindings/wasm/bindings.cpp

```diff
@@ -169,9 +169,9 @@ EMSCRIPTEN_BINDINGS(whatever) {
       .function("add", &Union)
       .function("subtract", &Difference)
       .function("intersect", &Intersection)
-      .function("getMesh", &Manifold::GetMesh)
+      .function("_GetMesh", &Manifold::GetMesh)
       .function("refine", &Manifold::Refine)
-      .function("_Warp", &Warp, allow_raw_pointers())
+      .function("_Warp", &Warp)
```

Comment:
`allow_raw_pointers` is not needed now? Is it documented in emscripten documentation?

Reply:
Actually, I noticed you didn't add it when you made `levelSet`, which uses the same thing; I believe you changed the pointer type from `void *`. So I figured I wouldn't need it here anymore either, and sure enough, it works.

Reply:
Oh OK, good to know